roles/confluent.kafka-broker/templates/server.properties.j2

```diff
@@ -1,5 +1,5 @@
 # Maintained by Ansible
-listeners=PLAINTEXT://:{{broker.config.port}}
+listeners=PLAINTEXT://{{broker.config.host}}:{{broker.config.port}}
```

Comment:
If not an empty string above, then `{{ broker.config.host | default "" }}` (or whatever the correct Jinja syntax is)?

Reply:
I vote for the first option, not to have hidden default values inside templates.